GVFS/GVFS.UnitTests/Common/GVFSLockTests.cs

```diff
@@ -0,0 +1,206 @@
+﻿using GVFS.Common;
+using GVFS.Common.NamedPipes;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Category;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Virtual;
+using NUnit.Framework;
+using System;
+
+namespace GVFS.UnitTests.Common
+{
+    [TestFixture]
+    public class GVFSLockTests : TestsWithCommonRepo
+    {
+        private static readonly NamedPipeMessages.LockData DefaultLockData = new NamedPipeMessages.LockData(1234, false, false, "git command");
+
+        [TestCase]
+        public void TryAcquireLockForExternalRequestor()

```

Comment:
The name of this test doesn't appear to match what it's doing (unless I'm missing something).

Should it be something like `ReleaseLockHeldByExternalProcess_WhenProcessDead`?

Reply:
This is the happy path for acquiring and releasing the lock and make sure it is free at the end.  Will rename to `TryAcquireAndReleaseLockForExternalRequestor`.

Reply:
Thanks! I misread what the test was doing in my last review.